z3c/dependencychecker/tests/test_package_overrides.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+from z3c.dependencychecker.package import Package
+from z3c.dependencychecker.dotted_name import DottedName
+import os
+
+
+EMPTY_FILE = ''
+IGNORE_OTHER_KEYS = """[servers]
+one = "1.2.3.4"
+"""
+ONLY_TOOL_TABLE = """[tool]
+two = "5.6.7.8"
+"""
+EMPTY_DEPENDENCY_TABLE_NESTED = """[tool]
+    [dependencycheker]
+"""
+EMPTY_DEPENDENCY_TABLE_MERGED = """[tool.dependencychecker]"""
+ONE_METAPACKAGE = """[tool.dependencychecker]
+Zope2 = ["Products.Five", ]
+"""
+MORE_METAPACKAGES = """[tool.dependencychecker]
```

Comment:
Do we only want metapackages under this key? No other configuration (`verbose = True`, for instance)?

Reply:
Mostly to keep it simple. I expanded the code a little bit to take care only about listings, so for options and other things we could have, yet another subtable, i.e. `[tool.dependencychecker.options]`.

Sounds that good?

As soon as we introduce other options/configuration we can revisit the approach.